roles/image_template/tasks/qcow2_image.yml

```diff
@@ -83,7 +83,8 @@
     - name: Check if template already exists
       ovirt_template_info:
         auth: "{{ ovirt_auth }}"
-        pattern: "name={{ template_name }} and datacenter={{ dc_info.ovirt_datacenters[0].name }}"
+        pattern: "name={{ template_name }} and datacenter={{ dc_info.ovirt_datacenters[0].name }}
+          and version_name={{ template_version_name | default('\"base version\"') }}"
```

Comment:
Please move the default value to [default yaml](https://github.com/oVirt/ovirt-ansible-collection/blob/master/roles/image_template/defaults/main.yml) or remove the default value as mentioned in previous comment, in which case you would need to check if the `template_version_name` is specified using the `{% if template_version_name is defined %} and version_name={{ template_version_name }} {% endif %}`